packages/babel-helper-create-regexp-features-plugin/src/util.ts

```diff
@@ -47,7 +47,8 @@ export function generateRegexpuOptions(node, features) {
   }
   return {
     useUnicodeFlag,
-    onNamedGroup: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    onNamedGroup: (name, index) => {},
```

Comment:
We can mark the return type and remove `name`, `index` here.

```typescript
{
    useUnicodeFlag: boolean,
    onNamedGroup: (name: string, index: number) => void,
    namedGroup: boolean,
    unicodePropertyEscape: boolean,
    dotAllFlag: boolean,
    lookbehind: boolean
};
```

Reply:
done